plugins/inputs/monit/status_response.xml

```diff
@@ -0,0 +1,216 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If you don't make each of these variables for your test and serve them, lets add these `xml` files into a sub-directory called something like `testdata`